recipes/protobuf/meta.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+    name: protobuf
+    version: "3.0.0a3"
```

Comment:
I only stumbled upon this because I was working with tensorflow yesterday which requires 3.0.0a3 and there was some concerns about shipping alpha software.  I guess a beta release might be less concerning.